### PR TITLE
Vagrant documentation improvement

### PR DESCRIPTION
**Pull request by atsag**
## Description

Added link to download an already-compiled version of the `wsk` binary (removing references to compiled wsk binaries in other OS's) and appropriate Windows Powershell code to correctly handle nested commands in Windows systems

## Related issue and scope
- [x] I opened an issue to propose and discuss this change (#4113)

## My changes affect the following components
- [ ] API
- [ ] Controller
- [ ] Message Bus (e.g., Kafka)
- [ ] Loadbalancer
- [ ] Invoker
- [ ] Intrinsic actions (e.g., sequences, conductors)
- [ ] Data stores (e.g., CouchDB)
- [ ] Tests
- [ ] Deployment
- [ ] CLI
- [ ] General tooling
- [x] Documentation

## Types of changes

- [x] Bug fix (generally a non-breaking change which closes an issue).
- [ ] Enhancement or new feature (adds new functionality).
- [ ] Breaking change (a bug fix or enhancement which changes existing behavior).

## Checklist:

- [x] I signed an [Apache CLA](https://github.com/apache/incubator-openwhisk/blob/master/CONTRIBUTING.md).
- [ ] I reviewed the [style guides](https://github.com/apache/incubator-openwhisk/wiki/Contributing:-Git-guidelines#code-readiness) and followed the recommendations (Travis CI will check :).
- [ ] I added tests to cover my changes.
- [ ] My changes require further changes to the documentation.
- [x] I updated the documentation where necessary.

### CODE REVIEW

File: tools/vagrant/README.md [hunk 1]
```diff
@@ -131,7 +135,10 @@ Vagrant installation is self-signed. Alternatively, you can configure your
 _apihost_ to use the non-SSL interface:
 
 ```
+# Non-Windows systems:
 wsk property set --apihost http://192.168.33.16:10001 --auth `vagrant ssh -- cat openwhisk/ansible/files/auth.guest`
+# Windows systems (Powershell):
+wsk property set --apihost http://192.168.33.16:10001 --auth $(vagrant ssh -- cat openwhisk/ansible/files/auth.guest)
```
Comment (csantanapr) [inline tools/vagrant/README.md]: Using $(....) should also work on Linux and Mac. 
Don’t have the distinction have the same command for all

File: tools/vagrant/README.md [hunk 2]
```diff
@@ -116,7 +116,11 @@ deployment and run the "echo" action again to test. The following commands
 assume that you have `wsk` setup correctly in your PATH.
 ```
 # Set your OpenWhisk Authorization Key.
+
+# Non-Windows systems:
 wsk property set --apihost 192.168.33.16 --auth `vagrant ssh -- cat openwhisk/ansible/files/auth.guest`
+# Windows systems (Powershell):
+wsk property set --apihost 192.168.33.16 --auth $(vagrant ssh -- cat openwhisk/ansible/files/auth.guest)
```
Comment (csantanapr) [inline tools/vagrant/README.md]: Using $(....) should work on Linux and Mac, have the same command for all

File: tools/vagrant/README.md (hunk 1 above)
Comment (atsag) [reply tools/vagrant/README.md]: OK, though changed the wording to imply that Powershell should be used when the deployment is hosted in Windows (as the default cmd prompt won't correctly substitute the result of expressions).

File: tools/vagrant/README.md (hunk 2 above)
Comment (atsag) [reply tools/vagrant/README.md]: OK, though changed the wording to imply that Powershell should be used when the deployment is hosted in Windows (as the default cmd prompt won't correctly substitute the result of expressions).

**Comment by rabbah** [comment]
LGTM Thanks for improving the docs @atsag.

**Comment by csantanapr** [comment]
restarting Travis build, we are having trouble with tests

**Pull request merged**